setup.cfg

```diff
@@ -1,46 +0,0 @@
-[check-manifest]
-ignore =
-    .pre-commit-config.yaml
-
-[metadata]
-name = pytest-rerunfailures
-version = 13.1.dev0
-url = https://github.com/pytest-dev/pytest-rerunfailures
-description = pytest plugin to re-run tests to eliminate flaky failures
-long_description = file: HEADER.rst, README.rst, CHANGES.rst
```

Comment:
Previously this meant we had these three joined together for the PyPI description.

We should figure out how to do it in `pyproject.toml`:

```toml
readme = ?
```

Reply:
there is a way to make it on the fly...
Define a function in a build_scripts.py file:

```python
# build_scripts.py

def generate_long_description():
    long_description = '''
    Your dynamic long description goes here. You can generate it on the fly using Python code.
    '''
    return long_description
```
Then in your pyproject.toml, you can refer to this function:

```toml
[project]
readme = "build_scripts.py::generate_long_description"
```

Reply:
https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#dynamic-metadata says we can do something like:

```toml
[project]
dynamic = ["readme"]

[tool.setuptools.dynamic]
readme = {file = ["README.rst", "USAGE.rst"]}
```

Reply:
added and check the created package and seems correct